### ISSUE TITLE
Sound from alarms/timers is played only once.

### ISSUE THREAD

**Issue by petcat30** [Bnyro/ClockYou#67]
### Steps to reproduce

Create alarm or timer.

### Expected behavior

When the time is up I expected multiple sounds, thus alarming me.

### Actual behavior

A single chime is played and easy to miss/not even hear.

### Clock You version

3.1

### Android version

13, GOS

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.
